frame/composable-traits/src/dex.rs

```diff
@@ -134,6 +134,30 @@ pub trait Amm {
 	) -> Result<Self::Balance, DispatchError>;
 }
 
+#[derive(Encode, Decode, MaxEncodedLen, TypeInfo, Clone, PartialEq, Eq, Copy, RuntimeDebug)]
+pub enum RewardPoolType {
+	LP,
+	PBLO,
+}
+
+impl Default for RewardPoolType {
+	fn default() -> Self {
+		RewardPoolType::PBLO
+	}
+}
+
+#[derive(
+	Encode, Decode, MaxEncodedLen, TypeInfo, Clone, Default, PartialEq, Eq, Copy, RuntimeDebug,
+)]
+pub struct StakingRewardPool<RewardPoolId> {
+	pub pool_id: RewardPoolId,
+	pub pool_type: RewardPoolType,
+}
+
+pub const MAX_REWARDS: u128 = 100_000_000_000_000_000_000_000_u128;
+pub const REWARD_PERCENTAGE: u32 = 10;
+pub const FIVE_YEAR_BLOCKS: u32 = 5 * 365 * 24 * 60 * 10;
```

Comment:
I think this should be `5 * 365 * 24 * 60 * 5`, since our block time is 12s?

Reply:
I think we need to calculate this based on the configured runtime block time.